sdk/monitor/azure-monitor-opentelemetry-exporter/setup.py

```diff
@@ -85,8 +85,8 @@
         "azure-core<2.0.0,>=1.28.0",
         "fixedint==0.1.6",
         "msrest>=0.6.10",
-        "opentelemetry-api~=1.21",
-        "opentelemetry-sdk~=1.21",
+        "opentelemetry-api~=1.24",
```

Comment:
Any reason for the opentelemetry bumps?

Reply:
Not specifically. Don't we always want to keep them updated? If not, I can wait unti I make the upstream autoinstrumentation changes.

Reply:
This is a min version. There is no reason to keep them "updated" if there is no change in upstream that was released that needs to be supported explicitly.

Reply:
changed back